omnibus_build.sh

```diff
@@ -9,11 +9,22 @@
 PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
 LOG_LEVEL=${LOG_LEVEL:-"info"}
+JMX_VERSION=${JMX_BRANCH:-"current"}
+export AGENT_BRANCH=${AGENT_BRANCH:-"master"}
 export OMNIBUS_BRANCH=${OMNIBUS_BRANCH:-"master"}
 export OMNIBUS_SOFTWARE_BRANCH=${OMNIBUS_SOFTWARE_BRANCH:-"master"}
 export OMNIBUS_RUBY_BRANCH=${OMNIBUS_RUBY_BRANCH:-"datadog-5.5.0"}
 export INTEGRATION_CORE_BRANCH=${INTEGRATION_CORE_BRANCH:-"master"}
 
+REMOTE_AGENT_REPO_RAW="https://raw.githubusercontent.com/DataDog/dd-agent"
+LOCAL_DD_AGENT="/dd-agent-repo"
+
+unset_jmx() {
+    echo "JMX provided by dd-agent in selected branch, JMX environment variables will be ignored."
```

Comment:
I'm not sure what to do with this. On one hand you're absolutely right. On the other hand, we should be able to build to build any dd-agent branch on cue. We'd have to upload all jmxfetch releases to the bucket, which isn't much of a problem, but the jarfiles are already part of the repo,... not sure.